src/main/java/Tokenizer.java

```diff
@@ -0,0 +1,17 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Tokenizer {
+    private final String newSplitValue = "";
+    private final String defaultSplitValue = ",|;";
```

Comment:
상수선언시 static도 붙여주는것이 관례이오니 다음 링크를 참고해주시면 감사하겠습니다 :)
```suggestion
    private static final String SPLIT_VALUE = "";
    private static final String DEFAULT_SPLIT_VALUE = ",|;";
```

https://djkeh.github.io/articles/Why-should-final-member-variables-be-conventionally-static-in-Java-kor/